docs/changelog.rst

```diff
@@ -3,6 +3,7 @@ Changelog
 =========
 
 * :bug:`9730` Fix the issue where the app breaks when the user clicks on "Ethereum Staking" in the blockchain balance summary on the dashboard.
+* :bug:`-` Addresses tracked in L2s but not in ethereum will be checked in the airdrop section.
```

Comment:
no need to mention ethereum since "L2" has been said earlier.

perhaps, **"L2-only addresses will now be included in airdrop eligibility checks."** or **"Fixed issue where addresses existing only on L2 networks were not being checked for airdrop eligibility."**